data/hai/hai reveal 1 intro.txt

```diff
@@ -938,6 +936,69 @@ ship "Geodesic Geocoris" "Geodesic Geocoris (Phylactery)"
 	description "This modified Geocoris has identical cargo capacity, but is the result of an alien technology being merged with a hybrid Human-Hai operating system. It has additional shield emitters in a geodesic pattern over its main cargo wings, and some kind of integrated system core which seems to have something of a life of its own. As such, some systems are built-in and unable to be removed. Nevertheless it is significantly more survivable than a traditional Geocoris and at least as useful."
 
 
+# The spike in interest leads to ladybugs appearing in the Hai Merchant (Human) fleets.	
+mission "Hai Leaks Response 4C"
+	landing
+	invisible
+	to offer
+		has "event: hai ladybug available"
+	on offer
+		event "humans adopt ladybugs" 20 30
+		fail
```

Comment:
Just call "humans adopt ladybugs" wherever you call "hai ladybug available".

Reply:
One adds them to shipyards, the other adds them to fleets.
This setup also lets them be added to fleets smoothly on any saves currently already having passed the point where  "hai ladybug available" is applied.
So I don't really see the problem?
Especially as this has the added bonus of there being a delay between availability and adoption.

Reply:
We've already done a lot worse to people who've already done some of these missions. I don't think this is worth adding multiple new missions that do nothing but call event based on if a different event has been called. It's needlessly complex.
I'm not suggesting you combine the contents of the events, just that you call them in the same MissionAction, and you can still set it on a delay when doing that, for example:
```
	on complete
		event "wanderers: the eye opens"
		event "wanderers: derecho mass production" 16
		event "wanderers: spera anatrusk colony" 40
```

Reply:
> We've already done a lot worse to people who've already done some of these missions.

I dispute that, actually. The effort has been made to keep the experience intact - it's not the fault of the missions that the game couldn't handle a null pointer when the missions looked logical.

Reply:
I gotta agree adding it to current saves doesnt matter